CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.2.0] - 2022-02-15
+
+### Added
+
+- Use tox, black, isort, flake8, and mypy to control quality of code.
+- New function that verifies whether a webmention is valid.
```

Comment:
I wonder if we shouldn't add a `#### New` level and also include the function name.

e.g.

```md
#### New

- `validate_webmention` function to verifying webmentions using [vouch](https://indieweb.org/Vouch)
```